docs/particles/functional.rst

```diff
@@ -3,18 +3,17 @@
 Functions
 *********
 
-In addition to the |Particle| class, the `~plasmapy.particles`
+In addition to the |Particle| class, the `plasmapy.particles`
 subpackage has a functional interface.
 
 .. _particles-func-symbols:
 
 Symbols and names
 =================
 
-Several functions in `~plasmapy.particles` return string representations
-of particles, including `~plasmapy.particles.atomic_symbol`,
-`~plasmapy.particles.isotope_symbol`, `~plasmapy.particles.ionic_symbol`,
-and `~plasmapy.particles.element_name`.
+Several functions in `plasmapy.particles` return string representations
+of particles, including |atomic_symbol|, |isotope_symbol|, |ionic_symbol|,
+and |element_name|.
```

Comment:
At https://plasmapy--1158.org.readthedocs.build/en/1158/particles/functional.html it seems like none of these form links except for `ionic_symbol`... except that one gets replaced by `is_stable`?!